### PR TITLE
docs: history support in kconnect proposal

### PR DESCRIPTION
**Pull request by richardcase**
**What this PR does / why we need it**:
Created a proposal to add history functionality to kconnect

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: docs/proposals/20200827-history.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+```
+
+When using the command to filter connections made to EKS you might do the following:
+
+```bash
+kconnect ls --provider eks -o table
+
+| ID                        | Providers | Identity | Alias       | Config     | Flags                      |
+|===========================|===========|==========|=============|============|============================|
+|MTIzNDU2Nzg5L215Y2x1c3RlcjE| EKS       | SAML     | dev-eks     | (default)  | username=bob@somewhere.com |
```
Comment (seankelly001) [inline docs/proposals/20200827-history.md:120]: Should we decode this base64 value to display a user friendly value?

File: docs/proposals/20200827-history.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+```bash
+# Add an alias to a history item
+kconnect alias add --id MTIzNDU2Nzg5L215Y2x1c3RlcjE --alias dev-eks
+
+# Delete an alias from a history item
+kconnect alias rm --id MTIzNDU2Nzg5L215Y2x1c3RlcjE
+
+# List all the aliases an their history ids
+kconnect alias ls
```
Comment (seankelly001) [inline docs/proposals/20200827-history.md:175]: A clear command would be useful as well:
kconnect alias clear
or
kconnect aliam rm --all

File: docs/proposals/20200827-history.md (hunk 1 above)
Comment (richardcase) [reply docs/proposals/20200827-history.md]: I have updated the proposal to include 2 different identifiers:

- **id** which is a unique history identifier
- **providerId** which is generated by the providers and is only meant to be understood by the provider

So to answer the queery i don't think we should decode it.

File: docs/proposals/20200827-history.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+|===========================|===========|==========|=============|============|============================|
+|MTIzNDU2Nzg5L215Y2x1c3RlcjE| EKS       | SAML     | dev-eks     | (default)  | username=bob@somewhere.com |
+|                           |           |          |             |            | idp-provider=GoogleApps    |
+|---------------------------|-----------|----------|-------------|------------|----------------------------|
+
+```
+
+
+#### Using the history
+
```
[the remark this answers is not in the document]
Comment (richardcase) [reply docs/proposals/20200827-history.md:130]: Thats a great idea @seankelly001 . I have updated the proposal to include the interactive aspect.

**Pull request merged**